llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -62,6 +62,31 @@ const StringMap<Decor> SpirvDecorMap = {
 };
 #undef SYCL_COMPILE_TIME_PROPERTY
 
+enum FloatControl {
```

Comment:
Would be good to have a link to the specification which defines those masks and opcodes for reference and to double-check everything

Reply:
Added link to specification where SPIRV opcodes are mentioned (below https://github.com/intel/llvm/pull/11591/files#diff-79e7dc5d3e16f48929c547ceca541b8d5c2053b1c0f5fec4921928852e396770R82-R85)
Masks aren't specified, this is implementation detail.

Reply:
Oh, so do I understand correctly that masks are part of the contract with Headers? If so, should we add a comment that they should stay in sync with a certain header file in the runtime?

Reply:
Yes, you are right. Added comment that these two places must be in sync.